docs/servo_power_test/src/dut_controller.ts

```diff
@@ -72,4 +75,32 @@ echo "stop"\n`;
   public async sendCancelCommand() {
     await this.dut.write(this.CANCEL_CMD);
   }
+  public async login() {
+    let isUserNameEntered = false;
+    await this.dut.write('\n');
+    await this.dut.write('\n');
+    for (;;) {
+      const dutData = await this.discardAllDutBuffer(100);
+      if (dutData.includes('localhost login:')) {
+        await this.dut.write('root\n');
+        isUserNameEntered = true;
+        continue;
+      }
+      if (dutData.includes('Password:')) {
+        if (!isUserNameEntered) {
+          await this.dut.write('\n');
+          continue;
+        }
+        await this.dut.write('test0000\n');
+        const result = await this.discardAllDutBuffer(1000);
+        this.ui.addMessageToConsole(result);
+        if (result.includes('Login incorrect')) {
+          isUserNameEntered = false;
+          continue;
+        }
+        return;
+      }
+      return;
```

Comment:
It may be a good idea to add a check that the login attempt was successful or not and notify the user when it failed. (it can be done in another pull-request)

Reply:
(feel free to close this conversation and submit the change if you want!)